src/plugins/discover/public/application/components/default_discover_table/_table_cell.scss

```diff
@@ -35,6 +35,24 @@
   &:focus &__filter {
     opacity: 1;
   }
+
+  .osdDescriptionListFieldTitle {
+    margin: 0 4px 0 0 !important;
+  }
+
+  /* stylelint-disable-next-line @osd/stylelint/no_modifying_global_selectors */
+  &.eui-textNoWrap {
+    width: 1%;
+  }
+}
+
+.osdDocTable__limitedHeight {
+  /* stylelint-disable-next-line value-no-vendor-prefix */
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 5;
```

Comment:
I had forgotten we had that setting. I will go honor that.

Reply:
As a compromise, i was thinking of dividing the number of `maxHeight` by `line-height` and using that. Lemme know if you oppose that. An arbitrary pixel value doesn't sound helpful at all.

Reply:
> An arbitrary pixel value doesn't sound helpful at all.

it does not. But we have an issue where this was being used by users. If we are bringing back the old table, i want to honor that for them, but the new datagrid will not honor that since its not a table and line count controls are more useful than pixel heights like you mentioned. Since this is just for the legacy table lets just go with that setting :)

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5783